lib/monitor.js

```diff
@@ -132,6 +132,25 @@ Monitor.prototype.stopSocketsMonitor = function () {
 };
 
 /**
+ * Capture stream data.
+ */
+Monitor.prototype.captureStream = function (streamName, stream) {
```

Comment:
`captureStream` seems misleading, since its' not actually catching data but just stream events... maybe `captureStreamEvents`?